pkg/cloudprovider/vm.go

```diff
@@ -113,8 +113,8 @@ func (c *VMClient) UpdateIdentities(rg, nodeName string, vm compute.VirtualMachi
 	if err = future.WaitForCompletionRef(ctx, c.client.Client); err != nil {
 		return fmt.Errorf("failed to wait for identity update completion for vm %s in resource group %s, error: %+v", nodeName, rg, err)
 	}
-	stats.UpdateCount(stats.TotalUpdateCalls, 1)
-	stats.Update(stats.CloudUpdate, time.Since(begin))
+	stats.Increment(stats.TotalPatchCalls, 1)
+	stats.Aggregate(stats.CloudPatch, time.Since(begin))
```

Comment:
Wouldn't using `Aggregate` here keep adding the `CloudPATCH` duration for all the base concurrent calls? The final statistics we print will have a sum of all Cloud PATCHs from the concurrent update.

Reply:
Right, let me change that.